platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ImportMigrationManagerImpl.java

```diff
@@ -141,8 +143,10 @@ public ImportMigrationManagerImpl(
       this.currentProductBranding = currentProductBranding;
       this.currentProductVersion = currentProductVersion;
       // process migratables' metadata
-      JsonUtils.getMapFrom(metadata, MigrationContextImpl.METADATA_MIGRATABLES)
+      JsonUtils.getMapFrom(metadata, MigrationContextImpl.METADATA_MIGRATABLES, false)
           .forEach((id, o) -> getContextFor(id).processMetadata(JsonUtils.convertToMap(o)));
+      customSystemProperties = this.getImportedCustomSystemProperties(metadata);
```

Comment:
✏️ I believe we can omit to re-read the custom system properties if they are not yet used in the import process.